src/Stripe.Portable/Stripe.Portable.csproj

```diff
@@ -0,0 +1,399 @@
+    </Compile>
+    <Compile Include="..\Stripe\Services\Refunds\StripeRefundCreateOptions.cs">
+      <Link>Services\Refunds\StripeRefundCreateOptions.cs</Link>
+    </Compile>
+    <Compile Include="..\Stripe\Services\Refunds\StripeRefundService.cs">
+      <Link>Services\Refunds\StripeRefundService.cs</Link>
+    </Compile>
+    <Compile Include="..\Stripe\Services\Refunds\StripeRefundUpdateOptions.cs">
+      <Link>Services\Refunds\StripeRefundUpdateOptions.cs</Link>
+    </Compile>
+    <Compile Include="..\Stripe\Services\StripeBankAccountOptions.cs">
+      <Link>Services\StripeBankAccountOptions.cs</Link>
+    </Compile>
+    <Compile Include="..\Stripe\Services\StripeCreditCardOptions.cs">
+      <Link>Services\StripeCreditCardOptions.cs</Link>
+    </Compile>
+    <Compile Include="..\Stripe\Services\StripeDateFilter.cs">
+      <Link>Services\StripeDateFilter.cs</Link>
+    </Compile>
+    <Compile Include="..\Stripe\Services\StripeListOptions.cs">
+      <Link>Services\StripeListOptions.cs</Link>
+    </Compile>
+    <Compile Include="..\Stripe\Services\StripeRequestOptions.cs">
+      <Link>Services\StripeRequestOptions.cs</Link>
+    </Compile>
+    <Compile Include="..\Stripe\Services\StripeService.cs">
+      <Link>Services\StripeService.cs</Link>
+    </Compile>
+    <Compile Include="..\Stripe\Services\StripeSourceOptions.cs">
+      <Link>Services\StripeSourceOptions.cs</Link>
+    </Compile>
+    <Compile Include="..\Stripe\Services\Subscriptions\StripeSubscriptionCreateOptions.cs">
+      <Link>Services\Subscriptions\StripeSubscriptionCreateOptions.cs</Link>
+    </Compile>
+    <Compile Include="..\Stripe\Services\Subscriptions\StripeSubscriptionService.cs">
+      <Link>Services\Subscriptions\StripeSubscriptionService.cs</Link>
+    </Compile>
+    <Compile Include="..\Stripe\Services\Subscriptions\StripeSubscriptionUpdateOptions.cs">
+      <Link>Services\Subscriptions\StripeSubscriptionUpdateOptions.cs</Link>
+    </Compile>
+    <Compile Include="..\Stripe\Services\Tokens\StripeTokenCreateOptions.cs">
+      <Link>Services\Tokens\StripeTokenCreateOptions.cs</Link>
+    </Compile>
+    <Compile Include="..\Stripe\Services\Tokens\StripeTokenService.cs">
+      <Link>Services\Tokens\StripeTokenService.cs</Link>
+    </Compile>
+    <Compile Include="..\Stripe\Services\Transfers\StripeTransferCreateOptions.cs">
+      <Link>Services\Transfers\StripeTransferCreateOptions.cs</Link>
+    </Compile>
+    <Compile Include="..\Stripe\Services\Transfers\StripeTransferListOptions.cs">
+      <Link>Services\Transfers\StripeTransferListOptions.cs</Link>
+    </Compile>
+    <Compile Include="..\Stripe\Services\Transfers\StripeTransferService.cs">
+      <Link>Services\Transfers\StripeTransferService.cs</Link>
+    </Compile>
+    <Compile Include="Extensions\BindingFlags.cs" />
+    <Compile Include="Extensions\ReflectionExtensions.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="packages.config" />
```

Comment:
Oh. They have to be separate because the Portable version requires a slightly different set of packages to be installed, with different platform targets.